### PR TITLE
Fix/yeremenko

### PR DESCRIPTION
**Pull request by Sergey988**

### CODE REVIEW

File: src/app/presentation/components/modal-invites/modal-invites.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
+  {{ userId }} sdfdsfsfs
```
Comment (YevheniiVovk) [inline src/app/presentation/components/modal-invites/modal-invites.component.html]: For a what reason we add 'sdfdsfsfs' ?

File: src/app/presentation/components/modal-invites/modal-invites.component.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  userName: string | undefined;
+
+  constructor(private authenticationService: AuthenticationService) { }
+
+
+  ngOnInit() {
+    if (this.authenticationService.currentUser.Id)
+      this.userId = this.authenticationService.currentUser.Id;
+  
+      console.log(this.userId);
```
Comment (YevheniiVovk) [inline src/app/presentation/components/modal-invites/modal-invites.component.ts]: Why we log userId into console. Remove it.

File: src/app/presentation/components/modal-invites/modal-invites.component.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  styleUrls: ['./modal-invites.component.css']
+})
+export class ModalInvitesComponent implements OnInit {
+
+  userId: string;
+  userName: string | undefined;
+
+  constructor(private authenticationService: AuthenticationService) { }
+
+
```
Comment (YevheniiVovk) [inline src/app/presentation/components/modal-invites/modal-invites.component.ts]: A lot of nested enters at current file. Cleanup it

File: src/app/presentation/components/user-header/user-header.component.css [hunk 4]
```diff
@@ -11,97 +11,72 @@
...
-.drop ul li:hover a{
+ ul.dropmenu li:hover a{
     background: #48537a;
     color: white;
+    
+}
+
+.test .mat-menu-panel {
+    background: #cf4343;
 }
```
Comment (YevheniiVovk) [inline src/app/presentation/components/user-header/user-header.component.css]: Add empty line to end of file

File: src/app/presentation/components/user-header/user-header.component.ts [hunk 5]
```diff
@@ -2,33 +2,40 @@ import { AuthenticationService } from './../../../core/services/authentication.s
...
     this.authenticationService.Logout().subscribe(()=>{
       this.router.navigate(['']);
     });
   }
 
+  modalInvites() {
+    const dialogRef = this.dialog.open(ModalInvitesComponent);
+
+    dialogRef.afterClosed().subscribe(result => {
+      console.log(`Dialog result: ${result}`);
```
Comment (YevheniiVovk) [inline src/app/presentation/components/user-header/user-header.component.ts]: Same as previous. Do we need a console log here ?

File: src/app/presentation/components/user-header/user-header.component.ts [hunk 6]
```diff
@@ -2,33 +2,40 @@ import { AuthenticationService } from './../../../core/services/authentication.s
...
 @Component({
   selector: 'app-user-header',
   templateUrl: './user-header.component.html',
   styleUrls: ['./user-header.component.css'],
   providers: [AuthenticationService]
 })
+
 export class UserHeaderComponent implements OnInit {
   
   userMessageCount = '2';
```
Comment (YevheniiVovk) [inline src/app/presentation/components/user-header/user-header.component.ts]: If you add const value please leave the comment that describe for a what reason and what it's mean

File: src/app/core/models/userInviteList.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+export class UserInvite {
+    date?: string;
+    isConfirm?: boolean;
+    workspaceName?: string;
+    fromUserName?: string;
```
Comment (YevheniiVovk) [inline src/app/core/models/userInviteList.ts]: Why all of this field are nullable ? 
Model should be same as server model

File: src/app/presentation/components/modal-invites/modal-invites.component.html [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+      <div class="invite-cart">
+        <div>
+          <p class="invite-title">{{item.workspaceName}}</p>
+          <p class="invite-description">
+            <span>From user:</span> <span class="font-weight-600">{{item.fromUserName}}</span>
+            <span>|</span>
+            <span>Date time:</span> <span class="font-weight-600">{{item.date}}</span>
+          </p>
+        </div>
+        <div *ngIf="item.isConfirm == null; else noButtons">
```
Comment (YevheniiVovk) [inline src/app/presentation/components/modal-invites/modal-invites.component.html:13]: Could server return null as a value of this field ?

File: src/styles.css [hunk 9]
```diff
@@ -59,17 +62,15 @@ body * {
...
-    /* background-color: #685e9d !important; */
-    background-color: #363446 !important;
-    
-    letter-spacing: 1px;
 }
-
-
-button:hover {
+.form-section button, .home-page button {
+    background-color: #363446 !important;
```
Comment (YevheniiVovk) [inline src/styles.css]: Could we found another way to don't use !important;

File: src/app/presentation/components/home/home.component.css [hunk 10]
```diff
@@ -143,5 +142,8 @@ button:hover{
     text-align: left;
 }
 .title-info a {
-    display: inline-flex !important;
+    display: inline-flex;
+}
+.home-title-color {
+    color: #cdcdcd;
 }
```
Comment (YevheniiVovk) [inline src/app/presentation/components/home/home.component.css]: Empty line of end file
Please add this

**Review by YevheniiVovk — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]
good job

**Pull request merged**